src/api/app/models/bs_request.rb

```diff
@@ -52,6 +52,8 @@ class SaveError < APIException
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
 
+  belongs_to :user, foreign_key: 'creator', primary_key: 'login'
```

Comment:
Because we expect everywhere that it is the login and not an object
https://github.com/openSUSE/open-build-service/search?l=Ruby&q=creator&type=Code&utf8=%E2%9C%93